_1327/information_pages/templates/information_pages_base.html

```diff
@@ -10,24 +10,28 @@
 {% endblock %}
 
 {% block sidebar %}
-	{{ toc|safe }}
 	<dl class="meta-information">
 		<dt>{% trans "Author" %}</dt>
 		<dd>{{ document.author }}</dd>
 		<dt>{% trans "Created" %}</dt>
-		<dd>{{ document.created }}</dd>
+		<dd>{{ document.created|date:"d. M Y, H:i" }}</dd>
 	</dl>
+	<div class="divider"></div>
 	{% block information_page_nav %}
 		{% get_obj_perms request.user for document as "document_perms" %}
 		{% if "change_informationdocument" in document_perms %}
-			<div class="divider"></div>
 			<a class="btn btn-sm btn-sidebar {% if active_page == 'view' %}btn-warning{% else %}btn-default{% endif %}" href="{% url 'information_pages:view_information' document.url_title %}" role="tab">{% trans "View" %}</a>
 			<a class="btn btn-sm btn-sidebar {% if active_page == 'edit' %}btn-warning{% else %}btn-default{% endif %}" href="{% url 'information_pages:edit' document.url_title %}" role="tab">{% trans "Edit" %}</a>
 			<a class="btn btn-sm btn-sidebar {% if active_page == 'attachments' %}btn-warning{% else %}btn-default{% endif %}" href="{% url 'information_pages:attachments' document.url_title %}" role="tab">{% trans "Attachments" %}</a>
 			<a class="btn btn-sm btn-sidebar {% if active_page == 'permissions' %}btn-warning{% else %}btn-default{% endif %}" href="{% url 'information_pages:permissions' document.url_title %}" role="tab">{% trans "Permissions" %}</a>
 			<a class="btn btn-sm btn-sidebar {% if active_page == 'versions' %}btn-warning{% else %}btn-default{% endif %}" href="{% url 'information_pages:versions' document.url_title %}" role="tab">{% trans "Versions" %}</a>
+			<div class="divider"></div>
 		{% endif %}
 	{% endblock %}
+	<div class="toc">
```

Comment:
``{{ toc|safe }}`` already includes a ``div#toc``. You might want to rename this one.